libnmstate/nm/nmclient.py

```diff
@@ -44,7 +43,15 @@ def client(refresh=False):
     # refresh is a workaround to get the current state when GMainLoop is not
     # running
     if _nmclient is None or refresh:
-        _nmclient = NM.Client.new(None)
+        if NM:
+            _nmclient = NM.Client.new(None)
+        else:
+            logging.error(
+                'Missing introspection data for libnm'
+                'please make sure to install it prior to running nmstate.\n'
+                'Check the documentation for more information.'
```

Comment:
IMHO we can add the path of the missing file here (`/usr/lib64/girepository-1.0/NM-1.0.typelib`) to make it easier for users to find the appropriate package instead of having to find the documentation first.

Reply:
I'm afraid each distribution has it in a different path. This is the path on Ubuntu: `/usr/lib/x86_64-linux-gnu/girepository-1.0`

Reply:
Do we actually have the information in the documentation?

Reply:
We are missing a lot of documentation.
It should have appeared here: https://nmstate.github.io/#install

I will add it.

Reply:
@tyll , I have pushed #306 in the README.